src/hooks/use-get-tags-accordions.tsx

```diff
@@ -3,9 +3,10 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-import React, { ReactElement, useMemo } from 'react';
-
-import { useTags, ZIMBRA_STANDARD_COLORS } from '@zextras/carbonio-shell-ui';
+import React, { ReactElement, useCallback, useMemo } from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
is the runSearch type still missing?

Reply:
I didn't checked it in this PR

Reply:
![Screenshot 2022-05-24 at 3 28 59 PM](https://user-images.githubusercontent.com/74703491/170005451-3b3e206c-2d48-44ba-9df4-01d3490599e7.png)
@zovomat  yes it's still there

Reply:
did you run an npm install? the latest version should have it

Reply:
the package have version 0.4.13 should i update it to 0.4.20 ?

Reply:
@zovomat yes

Reply:
@zovomat sorry I'm closing this PR because something went wrong while raising the pr and it is showing changes from previous tasks the new PR is https://github.com/Zextras/carbonio-calendars-ui/pull/77